src/styles/blocks/items.scss

```diff
@@ -0,0 +1,9 @@
+.items {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 942px;
```

Comment:
[CHECKLIST](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) 13th item, don't hardcode max-width, create variables for card-width, number of cards in a row and margins between them and calculate it based on that